<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotionControl</class>
 <widget class="QWidget" name="MotionControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MotionControl</string>
  </property>
  <widget class="QPushButton" name="forwardButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>forward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backwardButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>backward</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>207</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>速度：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="speedEdit">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>mm/s</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>right</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>207</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>转度：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="revolutionEdit">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>deg/s</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="railBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>导轨复位</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="rotateBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>转台复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rotateButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>旋转</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="leftBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>旋转至左侧</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="rightBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>旋转至右侧</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
